.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -8,7 +8,7 @@ body:
       label: "⚠️ Please check that this feature request hasn't been suggested before."
       description: "There are two locations for previous feature requests. Please search in both. Thank you. The **Label filters** may help make your search more focussed."
       options:
-        - label: "I searched previous [Ideas in Discussions](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/categories/ideas?discussions_q=category%3AIdeas+-label%3A%22status%3A+released%22+-label%3Aduplicate+sort%3Atop) didn't find any similar feature requests."
+        - label: "I searched previous [Ideas in Discussions](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/categories/ideas-any-new-feature-requests-go-in-issues-please) didn't find any similar feature requests."
```

Comment:
Thanks!

I tested this out, and when I go to the updated link, it does go to the correct category now, but doesn't include the extra search terms to filter and sort the results.

What I mean by this is, when I go the old link, the GitHub shows this text in the search box:

![image](https://user-images.githubusercontent.com/4840096/208516560-00fe9e35-4a0f-465b-8fb7-7f69ff838913.png)

Those extra search terms are:

`-label:"status: released" -label:duplicate sort:top `

There were intended to:

-  hide from users any things that had already been fixed
- hide any duplicates
- sort the most highly up-voted ideas at the top

So it's very close, but would it be possible to include the extra search options as well please?